FluentUI.Demo/src/main/res/color/fluent_default_icon_tint.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019.
```

Comment:
Looks like we should update all of the copyright format in the files being added to match the existing format? (which includes the MIT license line)

Reply:
(some files have no copyright header)